src/react-components/avatar-editor.js

```diff
@@ -163,7 +167,7 @@ export default class AvatarEditor extends Component {
   fileField = (name, label, accept, disabled = false, title) => (
     <div className={styles.fileInputRow} key={name} title={title}>
       <label htmlFor={`avatar-file_${name}`}>
-        <div className="img-box">{this.state.avatar.files[name] && <img src={this.state.avatar.files[name]} />}</div>
+        <div className="img-box" />
```

Comment:
This is for the file field, currently only used in the debug interface for the glb file. This one has a little square but no image. Made it its own function since I ended up adding stuff to the change handlers for imageFields